cmd/gctcli/commands.go

```diff
@@ -3789,7 +3789,7 @@ func getHistoricCandles(c *cli.Context) error {
 	} else {
 		exchangeName = c.Args().First()
 	}
-	if !validExchange(exchangeName) {
+	if exchangeName != "" && !validExchange(exchangeName) {
```

Comment:
we could probably do the exchangeName != "" check in validExchange()

Reply:
yeah left over from when i had support for empty exchanges will update it